.circleci/config.yml

```diff
@@ -3,28 +3,78 @@ jobs:
   lint:
     docker:
       - image: circleci/golang:1.12-node
-
     working_directory: /go/src/github.com/mattermost/mattermost-plugin-github
     steps:
       - checkout
-
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make check-style
 
   test:
     docker:
       - image: circleci/golang:1.12-node
-
     working_directory: /go/src/github.com/mattermost/mattermost-plugin-github
     steps:
       - checkout
-
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make test
 
+  build:
+    docker:
+      - image: circleci/golang:1.12-node
+    working_directory: /go/src/github.com/mattermost/mattermost-plugin-github
```

Comment:
Might be worth to move this over to a default `executor` like in https://github.com/kaakaa/mmplugin-announce-new-channel/blob/master/.circleci/config.yml#L2-L6. 2/5

Reply:
looks great this, will change